app/templates/home.html

```diff
@@ -1,10 +1,72 @@
-{% extends "base.html" %}
+{% extends "base.html" %} {% block content %}
 
-{% block content %}
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
```

Comment:
Please remove the jQuery from the project. Please remove and build this without jQuery.